CHANGELOG.md

```diff
@@ -0,0 +1,231 @@
+# v0.1.0 - 2022-10-28
+
+We’re on the board 🎉! This first release of iroh brings a new implementation of IPFS to the world. 
+
+Key things to highlight about the first release of iroh:
+
+* **Exchange Data with Kubo**. Iroh can interoperate with [kubo](https://github.com/ipfs/kubo) nodes on the IPFS network, pushing & fetching data.
+* **Service Oriented architecture**. Don't want p2p? Turn it off! Iroh will still work without it. This prepares us for
+* **Single CLI to control all services**. 
+* **Built for efficiency**. Iroh's memory footprint & CPU usage are either on par or better than the best interoperable IPFS implementations out there.
+* **Runs well on a laptop**. We have a custom installer script that will configure iroh for your laptop. See our [install docs](https://iroh.computer/docs/install)
+
+v0.1.0 ships as 4 binaries for Linux & macOS with zero external dependencies:
+* **iroh** - command line client
+* **iroh-store** - data storage daemon
+* **iroh-p2p** - peer 2 peer networking
+* **iroh-gateway** - IPFS-to-HTTP bridge
+
+You'll need to download all of them & put them on your $PATH to work with iroh. Our [install docs](https://iroh.computer/docs/install) have more info.
+
+Please do give iroh a try, we'd love to [hear your feedback](https://github.com/n0-computer/iroh/issues/new). Thanks!
+
+### Benchmarks
+We're runnning benchmarks on a relatively stock AWS box for ease-of-replication. We don't have benchmarks for IPFS network retrieval this round.
+
+### Adding Single Files
+Add a single file of different sizes with `iroh add --no-wrap --offline`:
```

Comment:
I think you can just say "Add a single file with `iroh add --no-wrap --offline`"

We get that it's different sizes from the chart